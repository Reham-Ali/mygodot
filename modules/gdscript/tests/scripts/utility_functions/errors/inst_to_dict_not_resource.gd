extends Resource


class NotResource:
	@export var number: int

func test():
	var obj:NotResource = NotResource.new()
	inst_to_dict(obj)
	print('not ok')
