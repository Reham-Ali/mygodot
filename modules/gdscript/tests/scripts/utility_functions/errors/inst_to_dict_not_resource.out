GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> utility_functions/errors/inst_to_dict_not_resource.gd
>> 9
>> Error calling GDScript utility function "inst_to_dict()": Not based on a resource file.
